update.py

```diff
@@ -112,6 +112,142 @@ def importNuitka():
 
 in_devcontainer = os.getenv("REMOTE_CONTAINERS_DISPLAY_SOCK") is not None
 
+FA_STYLE_MAP = {
+    "fas": "solid",
+    "far": "regular",
+    "fab": "brands",
+    "fal": "light",
+    "fad": "duotone",
+    "fat": "thin",
+    "fass": "sharp-solid",
+}
+
+FA_UTILITY_CLASSES = {
+    "fa",
+    "fa-fw",
+    "fa-spin",
+    "fa-pulse",
+    "fa-lg",
+    "fa-xs",
+    "fa-sm",
+    "fa-2x",
+    "fa-3x",
+    "fa-4x",
+    "fa-5x"
+}
+
+FA_SVG_PATH = Path("site") / "images" / "fontawesome"
```

Comment:
I really dislike using these rather than string paths, as they tend to always not work somewhere sometimes.

Reply:
I changed to mere Python